svthalia/concrexit
# Refactor events models to separate files

Closes #1063



### Summary
Refactor events models to separate files because the file was of immense size. And there would be no reason to not make it smaller, since it is easier to navigate between the models now. I have also renamed Registration to EventRegistration to prevent confusion with the registrations Registration model.

### How to test
Steps to test the changes you made:
1. Check that the whole events codebase still works aka we can start the server because imports should not have changed except for Registration to EventRegistration.
